src/components/AppNavigation/AppNavigationItem.vue

```diff
@@ -154,6 +154,7 @@ export default {
 		},
 		toggleCollapse() {
 			this.opened = !this.opened
+			this.item.opened = this.opened
```

Comment:
Maybe we should not use a local opened state then and use a computed + getter/setter instead ?

Reply:
Yes, otherwise this will also fail if the item is stored in vuex and changed without using a mutation.

Reply:
@juliushaertl directly forwarding a data from a vux store is not the best idea anyway.
So you will also need to use a getter/setter on your app as well for that. :)
Though using one for here as well seems like a good idea to avoir a double state (this?opened AND this.item.opened)

Reply:
With #195 apps will use this component directly, hence we can also just emit an event when the menu opens. Then apps can listen to that and update their state.

Reply:
Nice idea!
Let's use events and keep a local state then!